core/templates/dev/head/domain/exploration/InteractionObjectFactory.js

```diff
@@ -36,10 +37,19 @@ oppia.factory('InteractionObjectFactory', [
       return AnswerGroupObjectFactory.create(
         answerGroupDict.rule_specs, answerGroupDict.outcome, false);
     });
-
     return answerGroups;
   };
 
+  var generateFallbacksFromBackend = function(fallbackBackendDicts) {
+    if (fallbackBackendDicts !== undefined) {
```

Comment:
This isn't good ... it should never be undefined. I think this needs more investigation.